Description: ignore SDL_ACTIVEEVENT with SDL_APPINPUTFOCUS flag
 These events seem to come all the time, causing a pause which can't be
 unpaused
Bug-Debian: https://bugs.debian.org/909244
Author: Reiner Herrmann <reiner@reiner-h.de>

--- a/kobo.cpp
+++ b/kobo.cpp
@@ -1691,7 +1691,7 @@ void kobo_gfxengine_t::frame()
 			break;
 		  case SDL_ACTIVEEVENT:
 			// Any type of focus loss should activate pause mode!
-			if(!ev.active.gain)
+			if(!ev.active.gain && ev.active.state != SDL_APPINPUTFOCUS)
 				km.pause_game();
 			break;
 		  case SDL_QUIT:
